src/nanoemoji/bitmap_tables.py

```diff
@@ -25,22 +25,26 @@
 )
 from fontTools.ttLib.tables.C_B_D_T_ import cbdt_bitmap_format_17 as CbdtBitmapFormat17
 from functools import reduce
+from io import BytesIO
 from nanoemoji.config import FontConfig
 from nanoemoji.color_glyph import ColorGlyph
+from nanoemoji.png import PNG
+from nanoemoji.util import only
 from typing import (
     List,
     NamedTuple,
     Sequence,
     Tuple,
 )
+import statistics
```

Comment:
this seems unused?

Reply:
oops, I think I had a statistics.mean at some point and then removed it. Fixed.